index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello world!</h1>
```

Comment:
Just a suggestion - I would recommend adding a standard HTML markup and adding the header inside the page body.
But according to the task, this seems to be enough.